WcaOnRails/app/controllers/api/v0/api_controller.rb

```diff
@@ -69,7 +69,7 @@ def search(*models)
       render status: :bad_request, json: { error: "No query specified" }
       return
     end
-    result = models.flat_map { |model| model.search(query, params: params).limit(DEFAULT_API_RESULT_LIMIT) }
+    result = models.flat_map { |model| model.search(query, params: params).limit(DEFAULT_API_RESULT_LIMIT).as_json(api: true) }
```

Comment:
The actual purpose is to change what is included in the json exposed (ie: either the full body, in case we're in the posts page, or just a teaser/sticky status, in case we're on the home page).
Clearly the naming is not great, I'll rename this to `full_body`, that should make it more explicit!

Reply:
Argh, rather the other way around, because this specific line is posts-oblivious.